lib/notification-issue.coffee

```diff
@@ -137,7 +137,7 @@ class NotificationIssue
         resolve(@issueBody)
 
   normalizedStackPaths: (stack) =>
-    stack.replace /(^\W+at )([\w.]{2,} [(])?(.*)(:\d+:\d+[)]?)/gm, (m, p1, p2, p3, p4) => p1 + (p2 or '') +
+    stack?.replace /(^\W+at )([\w.]{2,} [(])?(.*)(:\d+:\d+[)]?)/gm, (m, p1, p2, p3, p4) => p1 + (p2 or '') +
```

Comment:
What's the point of this change?  I noticed the commit message said `fix copying report from string fatal`, but I'm not sure when stack can be undefined.

Reply:
`stack` is something that is set in the notification options. So it will be undefined if you create a fatal notification programmatically without setting the `stack` option

e.g.
```javascript
atom.notifications.addFatalError("fatal"); // no options set ∴ stack is undefined
```